app/src/main/java/com/xa/xpensauditor/SearchProduct.java

```diff
@@ -29,6 +29,9 @@
 import java.util.Arrays;
 import android.app.ProgressDialog;
 
+/**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.